elixir/apps/domain/priv/repo/migrations/20250725165751_add_psk_base_to_clients_and_gateways.exs

```diff
@@ -0,0 +1,35 @@
+defmodule Domain.Repo.Migrations.AddPskBaseToClientsAndGateways do
+  use Ecto.Migration
+
+  def up do
+    alter table(:clients) do
+      add(:psk_base, :binary, default: fragment("gen_random_bytes(128)"))
+    end
+
+    execute("UPDATE clients SET psk_base = gen_random_bytes(128) WHERE psk_base IS NULL;")
```

Comment:
Hmmm, I was going to use 128, but I think 32 is better to avoid exhausting the entropy pool.